bin/ghe-restore

```diff
@@ -161,6 +161,9 @@ hostname=$(echo "$GHE_HOSTNAME" | cut -f 1 -d :)
 # Show usage with no <host>
 [ -z "$GHE_HOSTNAME" ] && print_usage
 
+# Flag to indicate if this script has stopped Actions.
+ACTIONS_STOPPED=false
```

Comment:
@boxofyellow Init this variable here feels a bit arbitrary but I couldn't find a better place. Any idea? 

I also see that we `export` most of the variables but from what I see this is not needed here, should we do it anyway?

If we don't initialise the variable the `if $ACTIONS_STOPPED` will actually evaluate to `true`.

Reply:
I don't think we want to `export` it, since we are going we are only interested in using it here within this script.

Reply:
Hmmm that being said look what we are doing with `CLUSTER`

https://github.com/github/backup-utils/blob/20db671238089abd45ac32d4ef468639fee273ac/bin/ghe-restore#L198

Reply:
But that may be in place to support https://github.com/github/backup-utils/blob/master/share/github-backup-utils/ghe-restore-pages

Reply:
The `export CLUSTER` also confused me but the usage in `ghe-restore-pages` makes it clearer.